public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -305,6 +310,15 @@ mciModule.controller('PerformanceDiscoveryCtrl', function(
         field: 'test',
         _link: row => '/task/' + row.entity.taskId + '##' + row.entity.test,
         cellTemplate: 'ui-grid-link',
+        filter: {
+          noTerm: true,
+          condition: function(searchTerm, cellValue) {
+            if(vm.excludeCanaries && cellValue.match(/^(canary_.*|fio_.*|iperf.*|NetworkBandwidth)$/)) {
```

Comment:
Could we default this into the test filter instead? Having a separate checkbox makes it unclear how that interacts with the test filter